runtime/devnet/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "pop-runtime"
+name = "pop-runtime-devnet"
 version = "0.1.0"
 authors = { workspace = true }
 description = "A new Cumulus FRAME-based Substrate Runtime, ready for hacking together a parachain."
```

Comment:
Can we update this to the repo description? "Pop Network makes it easy for smart contract developers to use the Power of Polkadot."